cfme/tests/configure/test_access_control.py

```diff
@@ -1089,7 +1110,8 @@ def test_permissions_role_crud(appliance):
 
 
 @pytest.mark.tier(3)
-def test_permissions_vm_provisioning(appliance, provider):
+@pytest.mark.provider(classes=[VMwareProvider], selector=ONE)
```

Comment:
Do you need this provider specifically or any `InfraProvider` is ok? 
You may be able to use `pytest.mark.provider(classes=[InfraProvider], selector=ONE)`

Reply:
Any Infra Provider should be enough for permissions testing or tag testing, there's no VMWare-specific stuff.